src/realm/mixed.cpp

```diff
@@ -440,7 +441,6 @@ size_t Mixed::hash() const
 // LCOV_EXCL_START
 std::ostream& operator<<(std::ostream& out, const Mixed& m)
```

Comment:
This seems all fine, as long as we didn't rely on this previous format for json or anything. If we do need separate formats for json and query serialization, we can add an overload of `serializer::print_value(Mixed)` with this query format.